### ISSUE TITLE
Document using Materialize with Go

### ISSUE THREAD

**Issue by msudberg** [MaterializeInc/materialize#5187]
From [this thread](https://materializecommunity.slack.com/archives/C015KDVS7EV/p1609771052476500).

tl;dr: If you're using Materialize in a Go app, you can ingest matviews in other programming languages by using the standard postgres drivers for the language. pgx also works.

**Comment by rjnn** [comment]
In general, for all languages we should quickly pick out a set of drivers that we recommend. We should also have a clear statement about the perils of using ORMs on this page. Something similar to https://www.cockroachlabs.com/docs/v20.2/install-client-drivers.html

**Comment by awang** [comment]
Proposed terminology to describe these drivers: "community-tested libraries"

**Comment by benesch** [comment]
"Community tested" in, as in Materialize-hasn't-actually-tested-this? 😄

**Comment by awang** [comment]
yes -- to elaborate: we may have tested it, but we can't confidently 100% give it a seal of approval (and we may not be able to support it in a timely manner)

**Comment by bobbyiliev** [comment]
Pinning [Sequelize Library](https://sequelize.org/) (Node.js) as per the [Slack thread here](https://materializecommunity.slack.com/archives/C015KDVS7EV/p1641215838414000).

To get this package to work with Materialize Cloud there are a few extra parameters that need to be specified in the connection string. Mainly the `timezone` set to `UTC`, `clientMinMessages` set to `false` and under the SSL options, `rejectUnauthorized` set to `false`.

Here is an example snippet that works as expected:

```javascript
const { Sequelize, Model, DataTypes } = require('sequelize');
const fs = require('fs');

sequelize = new Sequelize('materialize', 'materialize', 'materialize', {
    timezone: 'UTC',
    clientMinMessages: false,
    host: '7luurjhv0d9.materialize.cloud',
    dialect: 'postgres',
    port: '6875',
    dialectOptions: {
        ssl: {
            require: true,
            rejectUnauthorized: false,
            key: fs.readFileSync('materialize.key').toString(),
            cert: fs.readFileSync('materialize.crt').toString(),
            ca: fs.readFileSync('ca.crt').toString(),
            encrypt: true
        }
    },
    retry :{
         match: [
             Sequelize.ConnectionError,
             Sequelize.ConnectionTimedOutError,
             Sequelize.TimeoutError,
             Sequelize.ConnectionAcquireTimeoutError,
             Sequelize.ConnectionRefusedError,
             /Deadlock/i,
             'SQLITE_BUSY'],
         max: 3
     }
 });

try {
    sequelize.authenticate();
    console.log('Connection has been established successfully.');
} catch (error) {
    console.error('Unable to connect to the database:', error);
}
```

**Comment by benesch** [comment]
Neat! We should get the SQL team's eyes on this, @bobbyiliev (cc @andrioni). Having to specifying `timezone` and `clientMinMessages` is definitely a bug on our end that I imagine would be easy to workaround. I'm weirded out by the `rejectUnauthorized: false` too; in theory setting the `ca` parameter should mean that the client can verify the server against the provided CA.

**Comment by bobbyiliev** [comment]
Hey @benesch, just tested this and indeed the `rejectUnauthorized: false` part is not needed. I must have left it there while testing yesterday. Good catch!

**Comment by morsapaes** [comment]
Linking to #10839. We should incorporate the output of this issue in the driver documentation.

**Comment by andrioni** [comment]
Also, @bobbyiliev: `clientMinMessages` is [now supported](https://github.com/MaterializeInc/materialize/pull/10707), so any valid values there should be accepted and work fine.

Checking the docs, timezone fails because they try to set it to `'+00:00'` by default: we could start accepting it as an input to fix that, since it's equivalent to UTC.

**Comment by morsapaes** [comment]
Since I'm at it, pinning [`vertx-pg-client`](https://vertx.io/docs/vertx-pg-client/java/). Maqqie reported [some issues](https://materializecommunity.slack.com/archives/C015KDVS7EV/p1617789775108000) a while back, but seem to be using it successfully since v0.7.2.
